infrastructure/conf/nginx-conf-deploy/default

```diff
@@ -229,6 +229,11 @@ server {
         rewrite "^(?:/[a-z]{2})?/directive-on-digital-talent(/.*|$)" /apps/web/dist/index.html break;
     }
 
+    # rewrite possible lang prefix then "skills" to apps/web index.hstml
+    location ~ "^(?:/[a-z]{2})?/skills(/.*|$)" {
+        rewrite "^(?:/[a-z]{2})?/skills(/.*|$)" /apps/web/dist/index.html break;
```

Comment:
Could you add the no-cache header to this match?  We want to make sure that clients are never caching index.html.

Reply:
Sounds good! Fixed by [913b728](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9614/commits/913b72862967a8ea89abfeb63f60b97be4f8389c)